update_ign.py

```diff
@@ -21,7 +21,19 @@
 #                     has four or more interfaces then you can choose to have bonded pairs for both the node and infra networks.
 #                     In either of those cases this script will have to be modified to account for those differences.
 
-# How to run this script: python3 update_ign.py
+#               we will have following option to choose from 
+#                    1.Single interface for node network and bond interface for infra network
+#                    2.Bond interface for both node and infra networks
+#                    3.Bond interface for node network and single interface for infra network
+#                    4.Single interface for both node and infra networks
+#
+#
+#               currently we support option one only(please update interface name in config.yaml file)
```

Comment:
> currently we support option one only

This comment is fine in the commit message but does not make sense here.  

> (please update interface name in config.yaml file)

This should be removed from the parentheses and clearly stated what changes need to be made in each case. So say something to the effect of:

    Please update the config.yaml as follows: